powerup.go

```diff
@@ -26,6 +26,7 @@ var (
 	re3       = regexp.MustCompile(`{{VEGETA}}`)
 	seed      = rand.NewSource(time.Now().UnixNano())
 	rando     = rand.New(seed)
+	phpay     = "PHPAY"
```

Comment:
hmm we should try and find a way to make this generic

Reply:
True, I could just splice out targets after they are returned. I'm gonna close this.